src/resolve/resolved-manager.c

```diff
@@ -444,13 +444,16 @@ static int determine_hostname(char **full_hostname, char **llmnr_hostname, char
 
 static const char *fallback_hostname(void) {
```

Comment:
rename the function please. it#s too confusing having fallback_hostname() and get_fallback_hostname() and they kinda do the same, but also don't

Reply:
The *other* function got renamed, so this name is non-confusing again.